hypha/static_src/src/javascript/apply/simplified.js

```diff
@@ -0,0 +1,19 @@
+(function () {
```

Comment:
`simplified` doesn't should like a semantic name to me. I think it could use better naming.  


Also, for accessibility, should handle the focus with the keyboard as well, and at this point, I would highly recommend using a library for this instead of using custom-made UI. 

https://a11y-guidelines.orange.com/en/web/components-examples/dropdown-menu/#

Reply:
>simplified doesn't should like a semantic name to me. I think it could use better naming.

Do you mean the file name?

 > I would highly recommend using a library for this instead of using custom-made UI.

I tried using bootstrap and it was working fine but it was changing other things as well like the fonts and all, the page was looking a bit weird/different from other pages. So, I moved forward with the custom UI. I think if we will move to a library we may need to update a few other things as well.

Reply:
Yes. name of the file and the css classes you are using. `simplified__dropbtn`. 

You can go with something like `dropdown.js` and html classes that start with `dropdown__`

Also, let's get into using a bulky library but I would encourage you to look at the implementation details of https://a11y-guidelines.orange.com/en/web/components-examples/dropdown-menu/#

Reply:
Added `dropdown.js` and `dropdown.css` as separate new files for dropdown only.